src/net/sourceforge/kolmafia/webui/RelayLoader.java

```diff
@@ -36,20 +39,32 @@ protected RelayLoader(final String location, final boolean isRelayLocation) {
     }
   }
 
+  private void waitForSVNUpdateToFinish() {
+    PauseObject pauser = new PauseObject();
+    int triesLeft = 6;
+    while ((triesLeft > 0) && LoginManager.isSvnLoginUpdateUnfinished()) {
```

Comment:
Why not make this a for loop like the loop below?

Reply:
The "loop below" was existing code that I didn't think about changing.  I tend to use a for when the number of times I do something is important and a while when there is some condition that I expect to change.  I can certainly make a decision and be consistent.  Stand by.